src/routes/webhooks/paddle.ts

```diff
@@ -100,6 +104,7 @@ export const updateUserSubscription = async ({
         createdAt: state ? data.data?.startedAt : null,
         subscriptionId: state ? data.data?.id : null,
         provider: state ? SubscriptionProvider.Paddle : null,
+        status: state ? UserSubscriptionStatus.Active : null,
```

Comment:
Ah I see now paddle will also have active status, what about old subs?

Reply:
I'll add it to existing ones